src/tests/ftest/object/same_key_different_value.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/python
+'''
+  (C) Copyright 2019 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+'''
+from __future__ import print_function
+
+import os
+import time
+import traceback
+from apricot import TestWithServers
+
+from daos_api import DaosPool, DaosContainer, DaosApiError
+
+class SameKeyDifferentValue(TestWithServers):
+    """
+    Test Description: Test to verify different type of values
+    passed to same akey and dkey.
+    :avocado: recursive
+    """
+    def setUp(self):
+        try:
+            super(SameKeyDifferentValue, self).setUp()
+
+            # parameters used in pool create
+            createmode = self.params.get("mode", '/run/pool/createmode/')
+            createsetid = self.params.get("setname", '/run/pool/createset/')
+            createsize = self.params.get("size", '/run/pool/createsize/')
+            createuid = os.geteuid()
+            creategid = os.getegid()
+
+            # initialize a python pool object then create the underlying
+            # daos storage
+            self.pool = DaosPool(self.context)
+            self.pool.create(createmode, createuid, creategid,
+                             createsize, createsetid, None)
+            self.pool.connect(1 << 1)
+
+            # create a container
+            self.container = DaosContainer(self.context)
+            self.container.create(self.pool.handle)
+
+            # now open it
+            self.container.open()
+
+        except DaosApiError as excpn:
+            print(excpn)
+            print(traceback.format_exc())
+            self.fail("Test failed during setup.\n")
+
+    def tearDown(self):
+
+        try:
+            if self.container:
+                self.container.close()
+
+            # wait a few seconds and then destroy
+            time.sleep(5)
+            if self.container:
+                self.container.destroy()
```

Comment:
This can be combined on above is block,
            if self.container:
                self.container.close()
                time.sleep(5)
                self.container.destroy()

Reply:
Done